build/cmake/functions.cmake

```diff
@@ -158,7 +160,7 @@ function(wx_set_common_target_properties target_name)
 endfunction()
 
 # Set common properties on wx library target
-function(wx_set_target_properties target_name is_base)
+function(wx_set_target_properties target_name is_base is_plugin)
```

Comment:
Could we perhaps use a single string, like `base`, `gui` or `plugin` instead of 2 booleans? Even though this is not C++, I still find calls to this function below passing it `FALSE TRUE` arguments perfectly unreadable...

Reply:
Yes, I'll an argument for this.